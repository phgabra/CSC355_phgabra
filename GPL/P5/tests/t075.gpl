// checking that math functions not typed as INT
double x1 = 100 * cos(40);
double x2 = 100 * sin(40);
double x3 = 100 * tan(40);
double x4 = 100 * atan(40);
double x5 = 100 * abs(40.123);
double x6 = 100 * abs(-40.123);
double x7 = 100 * sqrt(40);
