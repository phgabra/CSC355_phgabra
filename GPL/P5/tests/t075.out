gpl.cpp::main()
  input file(tests/t075.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double x1 = 76.6044
double x2 = 64.2788
double x3 = 83.91
double x4 = 8856.79
double x5 = 4012.3
double x6 = 4012.3
double x7 = 632.456
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
